pkg/tests/handlers/accounts.go

```diff
@@ -226,12 +227,23 @@ func TestClusterAuthorizations(ctx context.Context, options *types.TestOptions)
 	return nil
 }
 
+func randomizeResourceName(ResourceName_list [5]string) string {
+	ResourceName := fmt.Sprint(ResourceName_list[rand.Intn(len(ResourceName_list))])
+	return ResourceName
+}
+
+func randomizeCount(min_count int, max_count int) int {
+	Count := min_count + rand.Intn(max_count-min_count+1)
```

Comment:
I think the `+1` is not necessary here, with this you will always go from 5 to 11, instead of 4 to 10.

This is a good candidate to have a unit test, that checks that you number it's always between 4 and 10.